arrange-act-assert/README.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: pattern
+title: Arrange/Act/Assert
+folder: arrange-act-assert
+permalink: /patterns/arrange-act-assert/
+categories: Idiom
+tags:
+ - Unit Test
```

Comment:
We don't have an existing tag yet, but I would propose to use `Testing` here. Then we need to amend file https://github.com/iluwatar/java-design-patterns-web/blob/master/_data/tags.yml. See https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags.

Reply:
Changed in 882b569